it/firefox_relay_tos.md

```diff
@@ -1,7 +1,7 @@
 ﻿# Termini di servizio di Firefox Relay
 
-Versione 1.3, in vigore dal 10 marzo 2022
-{: datetime="2022-03-10" }
+Versione 1.4, in vigore dal 10 settembre 2022
```

Comment:
And obviously I missed the day in the one language I speak. WIll fix in a separate commit.